### PR TITLE
Switch to MIT license

### PR DESCRIPTION
**Pull request by jdhoffa**
Before merging this PR, we will need approval from all contributors.

### CODE REVIEW

**Review by maurolepore — CHANGES_REQUESTED** [review]
DESCRIPTION still mentions the old license. `usethis::use_mit_license()` should do the trick.

**Comment by maurolepore** [comment]
I approve changing to MIT license. 

For convenience I show here the other authors. Would you like me to contact them?

```
Authors@R: 
    c(person(given = "Mauro",
             family = "Lepore",
             role = c("aut", "cre", "ctr"),
             email = "maurolepore@gmail.com",
             comment = c(ORCID = "0000-0002-1986-7988")),
      person(given = "Jackson",
             family = "Hoffart",
             role = "aut",
             email = "jackson@2degrees-investing.org",
             comment = c(ORCID = "0000-0002-8600-5042")),
      person(given = "Klaus",
             family = "Hagedorn",
             role = "aut",
             email = "klaus@2degrees-investing.org"),
      person(given = "Florence",
             family = "Palandri",
             role = "aut",
             email = "florence@2degrees-investing.org"),
      person(given = "Evgeny",
             family = "Petrovsky",
             role = "aut"),
      person(given = "2 Degrees Investing Initiative",
             role = c("cph", "fnd"),
             email = "contact@2degrees-investing.org"))
```

**Comment by jdhoffa** [comment]
Yes, thanks Mauro, that would be great

**Comment by jdhoffa** [comment]
I approve the change to MIT license.

**Comment by maurolepore** [comment]
All authors mentioned here,

- [  ] @maurolepore 
- [  ] @jdhoffa 
- [  ] @2diiKlaus 
- [  ] @fpalandri
- [  ] @EvgenyPetrovsky
- [  ] 2 Degrees Investing Initiative (contact@2degrees-investing.org)


We plan to change the license of this package to MIT and we need all authors to support the change. For details contact @jdhoffa  or @AlexAxthelm. If you support the change, please write a comment in this thread ASAP, e.g. 

> I support changing the license of this package to MIT.

**Comment by maurolepore** [comment]
I support changing the license of this package to MIT.

**Comment by fpalandri** [comment]
I support changing the license of this package to MIT.

**Comment by fpalandri** [comment]
I support changing the license of this package to MIT.

**Comment by fpalandri** [comment]
I support changing the license of this package to MIT.

**Comment by 2diiKlaus** [comment]
I support changing the license of this package to MIT.

**Comment by EvgenyPetrovsky** [comment]
I approve changing to MIT license.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,2 @@
+YEAR: 2020
+COPYRIGHT HOLDER: 2 Degrees Investing Initiative France
```
Comment (maurolepore) [inline LICENSE:2]: Can you confirm we need the word "France"?

**Review by maurolepore — CHANGES_REQUESTED** [review]
Please run the helper from usethis, as I think a file is missing.

File: LICENSE (hunk 1 above)
Comment (jdhoffa) [reply LICENSE:2]: yes

**Review by maurolepore — APPROVED** [review]

**Comment by maurolepore** [comment]
Directors of 2DII investing confirmed they also support the change to MIT license. So we are ready to merge.

**Pull request merged**